anomaly_detector/config.py

```diff
@@ -78,6 +73,16 @@ class Configuration():
     # Maximum number of entries to be loaded for inference
     INFER_MAX_ENTRIES = 78862
 
+    # S3 credentials for storing model up to s3 post training.
+    S3_KEY = ""
+    S3_SECRET = ""
+    S3_HOST = ""
+    S3_BUCKET = ""
+    # local test dataset
+    LOCAL_DATA_FILE = ""
```

Comment:
so this should be name `LS_INPUT_PATH` right?

Reply:
@durandom yup I renamed that to the name you suggested.